docs-kits/kits/Industry Core Kit/Software Development View/page_notifications.mdx

```diff
@@ -0,0 +1,173 @@
+ - Content and structure of the property `content` are not stanardized and must be defined by the use case defining the notifications API. A schema for this property must be defined as part of the OpenAPI specification of the notification API of the use case.
+
+### Aspect Model "MessageHeaderAspect"
+
+The Catena-X Message Header contains standardized attributes for message processing that are common across several use cases. It targets a minimal set of aspects that are required for overall message handling, but not for the processing of the business related payload part. By standardizing the message header structure, we support a harmonized modelling approach and re-use of concepts.
+
+Identifier of Aspect Model:  `urn:samm:io.catenax.shared.message_header:3.0.0#MessageHeaderAspect` 
+
+Aspect model in GitHub:
+- Version 3.0.0: https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.shared.message_header/3.0.0
+
+#### Example: Aspect Model `MessageHeaderAspect 3.0.0`
+
+```json
+{
+  "header": {
+    "messageId": "urn:uuid:f9a97301-a000-44dd-b9d8-78488a40c6bb",
+    "context": "IndustryCore-DigitalTwinEventAPI-ConnectToParent:3.0.0",
+    "sentDateTime": "2024-07-05T08:13:33.20733Z",
+    "senderBpn": "BPNL000000000AAA",
+    "receiverBpn": "BPNL000000000ZZZ",
+    "expectedResponseBy": "2024-07-08T08:13:33.20733Z",
+    "version": "3.0.0"
+  },
+}
+```
+
+### Notification API
+
+The notification API of a use case must be defined with an OpenAPI specification. In this specification, all artefacts of the API are defined, e.g., name of the API, operation names, and notification data formats.
+
+#### Preconditions and Dependencies
+
+- The use case must standardize the notification API's name. The name of the notification API (used in property `http://purl.org/dc/terms/type`) must be defined in the Catena-X taxonomy published under https://w3id.org/catenax/taxonomy.
+- For a notification API, only the HTTP method POST is allowed.
+
+#### API Specification
+
+- Notification operations are modelled as endpoints (resources). For every operation, exactly one endpoint must be defined. This endpoint only supports the HTTP POST method.
+- The path URL should be the operation name with optional "-" as word separator. 
+- The operationId must be the notification operation name. The same name should be used as `<object>` in the `context` property of the notification header. This redundancy is recommended to allow the notification receiver to freely choose what technology to use for the backend service, e.g, a REST API or message-queue technology.
+- The notification is the actual request body of the HTTP POST method. Its schema must also be defined in the specification.
+  
+**Example:** [OpenAPI specification for Digital Twin Event API](https://eclipse-tractusx.github.io/api-hub/eclipse-tractusx.github.io/3.0.0/example-digital-twin-event-api.yaml) 
```

Comment:
Can you please change the link to a relative link instead of the `https://eclipse-tractusx.github.io` domain?